ui/src/jobs/components/ReferenceLetters.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Map } from 'immutable';
+import InlineUL from '../../common/components/InlineList/InlineUL';
+import URLList from '../../common/components/URLList';
+import EmailList from './EmailList';
+
+class ReferenceLetters extends Component {
+  render() {
+    const { referenceLetters } = this.props;
+    const urls = referenceLetters.get('urls', null);
+    const emails = referenceLetters.get('emails', null);
```

Comment:
`const shouldRender = urls || emails` 

that way you get rid of warnings that may appear when you remove default `null` for `get`.